src/components/AppNavigation/CalendarList/Trashbin.vue

```diff
@@ -236,6 +236,11 @@ export default {
 	max-width: 40vw;
 	margin: 2vw;
 }
+::v-deep .modal-wrapper .modal-container {
```

Comment:
```suggestion

::v-deep .modal-wrapper .modal-container {
```

Make linter and CI happy